packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group-item/gux-avatar-group-item.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  Component,
+  h,
+  JSX,
+  Prop,
+  Element,
+  State,
+  Listen,
+  Method
+} from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { logWarn } from '@utils/error/log-error';
+import { GuxAvatarAccent } from './gux-avatar-group-item.types';
+import {
+  groupKeyboardNavigation,
+  resetFocusableSiblings
+} from '../gux-avatar-group.service';
+
+/**
+ * @slot image - Avatar photo.
+ */
+
+@Component({
+  styleUrl: 'gux-avatar-group-item.scss',
+  tag: 'gux-avatar-group-item-beta',
+  shadow: true
+})
+export class GuxAvatarGroupItem {
+  private buttonElement: HTMLButtonElement;
+  private parentElement: HTMLGuxAvatarGroupBetaElement;
+
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Name which is shown as initials. Should be formatted 'Lastname Firstname' for JA, zhCN and KO names.
+   * Names without blank space will show first 2 characters of string.
+   */
+  @Prop()
+  name!: string;
+
+  /**
+   * Manually sets avatar accent
+   */
+  @Prop()
+  accent: GuxAvatarAccent = 'auto';
+
+  /**
+   *  Tracks the item index in group to faciliate auto accent colours
+   */
+  @State()
+  index: number = 0;
+
+  /**
+   * Removes mask from avatar display if is last in group
+   */
+  @State()
+  lastItemInGroup: boolean = false;
+
+  @State()
+  taxIndexVal: number = -1;
```

Comment:
Is this being used ?

Reply:
Yes, it's used to set the initial tab index and the index when clicked.

Reply:
minor typo on it `tabIndexVal`